packages/examples/examples/rollup/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "rollup-snap",
+  "version": "0.0.0",
+  "private": true,
+  "description": "An example Snap built using TypeScript and Rollup",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/snaps-skunkworks.git"
+  },
+  "license": "ISC",
+  "main": "src/index.js",
+  "files": [
+    "dist/",
+    "snap.manifest.json"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "manifest": "mm-snap manifest --fix",
+    "eval": "mm-snap eval -b dist/snap.js",
+    "serve": "concurrently \"rollup --config --watch\" \"mm-snap serve\""
```

Comment:
Is `--config` needed? 🤔

Reply:
Yeah, without it it doesn't use the config file.